pkg/gits/gitlab.go

```diff
@@ -172,7 +172,7 @@ func (g *GitlabProvider) GetRepository(org, name string) (*GitRepository, error)
 	if err != nil {
 		return nil, err
 	}
-	project, response, err := g.Client.Projects.GetProject(pid)
+	project, response, err := g.Client.Projects.GetProject(pid, nil, nil)
```

Comment:
these _nil_ values are needed now due to the upgrade of the GitLab client?

Reply:
The method signature for GetProject (plus a few others) changed after version 0.6.0 to GetProject(pid interface{}, opt *GetProjectOptions, options ...OptionFunc)  - neither the second or third parameters seemed to be required (and are nil-checked as such), do anything specifically useful in the context of use by jx currently.

Reply:
thanks